README.md

```diff
@@ -41,7 +45,7 @@ y = 10;
 
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
-<!-- Replace this comment with your answer -->
+The Value of x and y are 10. The variables do not change but that values do.
```

Comment:
Not quite. `x = y` does not form any sort of persistent relationship between `x` and `y`; instead, `y` is evaluated first, and the value of `y` gets assigned to `x`. So although `y` becomes 10 in the last line, `x` doesn't change!